src/components/DatePicker/DatePicker.vue

```diff
@@ -110,6 +113,9 @@ export default class DatePicker extends Vue {
   @Watch('dateText')
   @Emit('emitDateSync')
   private emitDateSync (date: string): string { return this.dateText }
+
+  @Watch('initialValue', { immediate: true })
```

Comment:
Since this watcher is only ever used when the component is created (ie, immediately), could you move this assignment right into line 91?

Reply:
i'll give it a try.  It makes sense I think.

Reply:
doesn't seem like i can do a direct assignment of the prop `initialValue` on line 91.  i imagine vue forces you to do assignment to another variable within watchers or other lifecycle hooks like `created`.  did you want me to do the initial assignment in `created`?

Reply:
Sure, good idea.

Reply:
added `created` hook and assignment of `initialValue` to `dateText` is done there now.